src/StudyUtils.in.jsm

```diff
@@ -1,5 +1,27 @@
 "use strict";
 
+/**
+* OUTSTANDING QUESTIONS FOR GREGG
+*  - shield-studies-addon schema description references attributes sent as 
+*  Map(string, string). Can you explain what this means?
```

Comment:
an object with string keys and string values.

Reply:
Ok thanks; do you explain anywhere what the attributes key is used for? I know it's for study-specific, string-string, key-value pairs -- looking at [Kamyar's CFR study](https://github.com/raymak/cfr-focused-release/blob/master/TESTPLAN.md), it looks like they are used to capture different, feature-related state information?